### PR TITLE
OGN tooltip

### PR DESCRIPTION
**Pull request by mek32390**
First pull request? Read our [guide to contributing](http://docs.originprotocol.com/#contributing)

### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [x] Wrap any new text/strings for translation
- [ ] ~~Map any new environment variables with a default value in the Webpack config~~
- [ ] ~~Update any relevant READMEs and [docs](https://github.com/OriginProtocol/docs)~~

### Description:

This is a new clean PR for the OGN tooltip. OGN balance still needs to be passed in to the tooltip from Redux. cc: @micahalcorn

### CODE REVIEW

**Comment by micahalcorn** [comment]
Mockup: https://app.zeplin.io/project/59fa2311bac7acbc8d953da9/screen/5b7538aa755a0b576b86c981

File: src/components/wallet-card.js [hunk 1]
```diff
@@ -23,6 +24,30 @@ class WalletCard extends Component {
...
+        id: '_wallet-card.balanceText',
+        defaultMessage: 'Having OGN is not required but will allow you \
+        to create a listing that will be more visible to buyers.'
+      },
+      getOGN: {
+        id: '_wallet-card.getOgn',
+        defaultMessage: 'Get OGN'
+      },
+      reccommendation: {
+        id: '_wallet-card.reccommendation',
```
Comment (Natasha08) [inline src/components/wallet-card.js]: `recommendation` here and line above

File: src/components/wallet-card.js [hunk 2]
```diff
@@ -282,4 +360,4 @@ const matchDispatchToProps = dispatch => ({
 export default connect(
   mapStateToProps,
   matchDispatchToProps
-)(WalletCard)
+)(injectIntl(WalletCard))
```
Comment (Natasha08) [inline src/components/wallet-card.js]: needs an extra line here

**Comment by micahalcorn** [comment]
@Natasha08 I haven't taken the time to look at this closely. Is there a more React-y way of handling the events rather than such heavy jQuery?

**Comment by Natasha08** [comment]
@mek32390 
Manually testing from the listings page, when I click on a listing => mouseover the tooltip balance heading I get this 
cursor_and_origin_protocol_demo_dapp

**Comment by micahalcorn** [comment]
> @mek32390
> Manually testing from the listings page, when I click on a listing => mouseover the tooltip balance heading I get this
> cursor_and_origin_protocol_demo_dapp

Sounds like we need to condition the rendering based on the parent component?

**Comment by Natasha08** [comment]
@micahalcorn I think that [React's Event System](https://reactjs.org/docs/events.html) could be used here instead of JQuery, and the component can determine whether or not to render things (like the tooltip)

**Comment by micahalcorn** [comment]
This still needs some fixes for the UX but I went ahead and merged it. We at least need to fix the hide delay so that the "Learn More" can be clicked. We probably also want it to appear programmatically and remain visible until dismissed by the user. @Natasha08 does it make sense to just go ahead and move all our tooltips away from jQuery via some established library?

**Pull request merged**